doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_debugging.rst

```diff
@@ -14,7 +12,7 @@ The main recommended tool for debugging in the |NCS| for the limited sampling of
 When working from the command line, you can use west with the GDB tool.
 For details, read the :ref:`Debugging with west debug <zephyr:west-debugging>` section on the :ref:`zephyr:west-build-flash-debug` page in the Zephyr documentation.
```

Comment:
Can you link to https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/test_and_optimize/debugging.html here?